app.js

```diff
@@ -14,6 +14,8 @@ let config = require('./config.json');
 let app = express();
 let logger = new Logger();
 
+console.log("__̴ı̴̴̡̡̡ ̡͌l̡̡̡ ̡͌l̡*̡̡ ̴̡ı̴̴̡ ̡̡͡|̲̲̲͡͡͡ ̲▫̲͡ ̲̲̲͡͡π̲̲͡͡ ̲̲͡▫̲̲͡͡ ̲|̡̡̡ ̡ ̴̡ı̴̡̡ ̡͌l̡̡̡̡.___");
```

Comment:
What's this?

Reply:
Nothing important, instead of a message saying the server has started I put some ascii art in there.